src/tests/ftest/server/replay.py

```diff
@@ -104,7 +104,10 @@ def verify_snapshots(self, container, expected):
             TestFail: if the detected list of snapshots does not match the detected list
         """
         self.log.debug("Expected list of snapshots: %s", expected)
-        detected = container.list_snaps()
+        detected = []
+        data = container.list_snaps()
+        for item in data['response']:
+            detected.append(item['epoch'])
```

Comment:
FYI, this is how I've been using it
```suggestion
        detected = [entry["epoch"] for entry in self.container.list_snaps()["response"]]
```
But that does beg the question of whether we should have a "simple" return, if that's what tests want